hack/run_test

```diff
@@ -372,15 +376,19 @@ function run_master_restart_test() {
   local PASS=master
 
   echo "Testing failed master restart"
-  local cluster_args="-e POSTGRESQL_ADMIN_PASSWORD=pass -e POSTGRESQL_MASTER_USER=$PGUSER -e POSTGRESQL_MASTER_PASSWORD=$PASS"
   local cid_suffix="mrestart"
+  local network=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
>i.e. why we can't just use cid_suffix

My thoughts behind this was running the same test case for different versions of postgresql in parallel on a single machine (which I think is what the RHEL CI is doing?). So we need something more random than the test case's suffix ID to identify each private network.

Will add a bit of commentary about this to the of code.